README.md

```diff
@@ -6,22 +6,10 @@ The Tank Trouble
 
 ## Inputs
 
-### Player 1
-
-| Key           | Description    |
-| ------------- | -------------- |
-| `arrow up`    | move forward   |
-| `arrow down`  | move backwards |
-| `arrow left`  | rotate left    |
-| `arrow right` | rotate right   |
-| `space`       | shoot bullet   |
-
-### Player 2
-
-| Key | Description    |
-| --- | -------------- |
-| `w` | move forward   |
-| `s` | move backwards |
-| `a` | rotate left    |
-| `d` | rotate right   |
-| `q` | shoot bullet   |
+| Movement       | Player 1      | Player 2 | Player 3 | Player 4 |
```

Comment:
Achas que ficou fixe assim?

(Fiz assim, só para não estar a repetir 4 vezes a mesma tabela)

Reply:
It looks a lot better now 💯 

However, I would rename `Kp` to `keypad` (all lower case to keep consistency) for better readability.
What do you think?

Reply:
I just put that because in the Godot, the name is `kp`, but yeah, I will change